provider/database.py

```diff
@@ -37,6 +37,7 @@ class Database:
 
     filters_design_doc_id = '_design/filters'
     only_triggers_view_id = 'only-triggers'
+    by_worker_view_id = 'by-worker'
 
     instance = os.getenv('INSTANCE', 'messageHubTrigger-0')
     canaryId = "canary-{}".format(instance)
```

Comment:
I removed the worker id from the canary doc id. i originally added it to avoid conflict between multiple workers, but it is not needed as updating 'instance' for each additional worker would suffice